### PR TITLE
Tech debt bug fix

### PR DESCRIPTION
**Pull request by Sushmareddy-OSI**

### CODE REVIEW

File: app/javascript/views/addUser/AddVerifiedUser.jsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ const AddNewUser = ({
             </Cards>
           </div>
         </div>
-      ) : verifiedUserDetails.verification_passed === false ? (
+      ) : verification_passed === false ? (
```
Comment (kelfink) [inline app/javascript/views/addUser/AddVerifiedUser.jsx]: should be able to refactor the to simply an else (:) clause without the comparison to ===

File: app/javascript/views/addUser/AddVerifiedUser.jsx [hunk 2]
```diff
@@ -8,17 +8,17 @@ import { formatPhoneNumberWithExt } from '../../_utils/formatters';
...
+  let newUserDetails = verifyNewUserDetails
+    ? verifyNewUserDetails.verifiedUserDetails
+    : {};
+  let verification_passed = newUserDetails
+    ? newUserDetails.verification_passed
+    : '';
   return (
     <div>
-      {verifiedUserDetails.verification_passed ? (
+      {verification_passed === true ? (
```
Comment (kelfink) [inline app/javascript/views/addUser/AddVerifiedUser.jsx]: should just be able to ask if verification_passed ?  without comparison against ===

File: app/javascript/views/addUser/AddVerifiedUser.jsx [hunk 3]
```diff
@@ -8,17 +8,17 @@ import { formatPhoneNumberWithExt } from '../../_utils/formatters';
...
-  email,
-  racfid,
-}) => {
-  const { verifiedUserDetails } = verifyNewUserDetails;
+const AddNewUser = ({ onCancel, onSave, verifyNewUserDetails }) => {
+  let verify = verifyNewUserDetails ? verifyNewUserDetails.verify : {};
+  let newUserDetails = verifyNewUserDetails
+    ? verifyNewUserDetails.verifiedUserDetails
+    : {};
+  let verification_passed = newUserDetails
```
Comment (kelfink) [inline app/javascript/views/addUser/AddVerifiedUser.jsx]: As long as you're assigning verification_passed with an expression used in a ternary, force it to be either true or false in this assignment.  No ''.  I think there' son use for an empty string here.

Comment (kelfink) [reply app/javascript/views/addUser/AddVerifiedUser.jsx]: I think these assignments (12-16) can be to 'const' variables

File: app/javascript/selectors/validateNewUserSelector.test.js [hunk 4]
```diff
@@ -4,24 +4,16 @@ describe('selectors', () => {
...
         last_name: 'third',
+        roles: [],
+      };
+      expect(selectNewUserRecords(state)).toEqual({
         roles: ['CWS-worker'],
       });
     });
 
-    it('roles set to default value  when verifyNewUserDetails are not available', () => {
+    it('roles set to default value  when New User are not available', () => {
```
Comment (RamuKammagani-osi) [inline app/javascript/selectors/validateNewUserSelector.test.js]: Need to rephrase this sentence!

**Pull request merged**